### ISSUE TITLE
Ability to delete tags? Not sure if we've covered this

### ISSUE THREAD

**Issue by kyleparry** [Japan-Digital-Archives/Japan-Digital-Archive#645]

**Comment by nickkapur** [comment]
@kyleparry Tags can be deleted on the backend through the admin interface. Do you mean deleting them on the front end? We discussed this at some point, and decided against allowing it.

**Comment by kyleparry** [comment]
Yes - do mean front-end. Is the idea to protect tags? I am just wondering about errors of entry, whether typos or mistranslations or misidentification. Seems like if you are logged in you should at least be able to delete tags you've generated - but I imagine yoking user IDs to individuals tags isn't so pretty.

**Comment by ebensing** [comment]
yah, they currently are not linked to userIds and supporting such a system would be a little annoying given the current technology on the site.
